### PR TITLE
move subprocess command to configuration, make subprocess to run async

### PR DESCRIPTION
**Pull request by shokakucarrier**

### CODE REVIEW

**Comment by shokakucarrier** [comment]
@ligangty as REs requested, command to actually sign the file should be configurable

File: config/charon.yaml [hunk 1]
```diff
@@ -12,6 +12,8 @@ ignore_signature_suffix:
   npm:
     - "package.json"
 
+detach_signature_command: "rpm-sign --detach-sign --key {{ key }} {{ file }}"
```
[the remark this answers is not in the document]
Comment (shokakucarrier) [reply config/charon.yaml:15]: REs want to change this, add pipe "|" or change the command all together
they think hard code command was a bad idea

File: quarkus-2.7.6.errors.log [hunk 2]
```diff
@@ -0,0 +1,8 @@
+2023-05-24 19:13:26,073 - WARNING - Warning: No manifest bucket is provided, will ignore the process of manifest uploading
+
+2023-05-24 19:13:26,338 - WARNING - Warning: No manifest bucket is provided, will ignore the process of manifest uploading
+
+2023-05-24 19:13:37,443 - WARNING - Warning: No manifest bucket is provided, will ignore the process of manifest uploading
+
+2023-05-24 19:13:37,720 - WARNING - Warning: No manifest bucket is provided, will ignore the process of manifest uploading
+
```
[the remark this answers is not in the document]
Comment (shokakucarrier) [reply quarkus-2.7.6.errors.log]: removed

File: config/charon.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (ligangty) [reply config/charon.yaml:15]: I know the want flexibility here. I mean if this command will be changed to something else, which will have more or less variables, not just {{key}} and {{file}}?

[the remark this answers is not in the document]
Comment (shokakucarrier) [reply config/charon.yaml:15]: they will have less, these two are the only thing they want, and even {{key}} can be replaced by pre configured value

**Review by ligangty — APPROVED** [review]
LGTM

**Pull request merged**